db/migrate/20181004180225_add_snp_annotations_to_phylo_trees.rb

```diff
@@ -0,0 +1,14 @@
+require 'open3'
+
+class AddSnpAnnotationsToPhyloTrees < ActiveRecord::Migration[5.1]
+  def change
+    add_column :phylo_trees, :snp_annotations, :string
+
+    PhyloTree.all.each do |pt|
+      s3_path = pt.s3_outputs["snp_annotations"]["s3_path"]
+      if Open3.capture3("aws", "s3", "ls", s3_path)[2].success?
```

Comment:
we should put this Open3.capture3("aws", "s3", "ls", s3_path)[2].success? into a more robust function.

Reply:
moved to PipelineRunsHelper for future iteration